pkgs/development/python-modules/ledgerblue/default.nix

```diff
@@ -11,7 +11,12 @@ buildPythonPackage rec {
     sha256 = "3969b3c375c0f3fb60ff1645621ebf2f39fb697a53851620705f27ed7b283097";
   };
 
-  buildInputs = [ hidapi pycrypto pillow protobuf future ecpy ];
+  buildInputs = [
```

Comment:
Why `buildInputs`?

Reply:
I just added my new dependencies where the others already were. Yes, it seems that it should be `propagatedBuildInputs`. I'll fix. Thanks.